core/admin/admin-notices.php

```diff
@@ -376,10 +377,49 @@ private function notice_experiment_promotion() {
 		return true;
 	}
 
-	public function print_admin_notice( array $options ) {
+	private function notice_design_not_appearing() {
+		$notice_id        = 'design_not_appearing';
+		$installs_history = get_option( 'elementor_install_history', [] );
+		$is_first_install = 1 === count( $installs_history );
+
+		if ( $is_first_install || ! current_user_can( 'update_plugins' ) || User::is_user_notice_viewed( $notice_id ) ) {
+			return false;
+		}
+
+		$options = [
+			'title' => esc_html__( 'Design not appearing as expected?', 'elementor' ),
+			'description' => esc_html__( 'Unexpected changes may occur after you update the plugin. Don’t worry - we’ve collected all the fixes for troubleshooting common issues.', 'elementor' ),
+			'id' => $notice_id,
+			'button' => [
+				'text' => esc_html__( 'Find a solution', 'elementor' ),
+				'url' => 'https://go.elementor.com/wp-dash-changes-do-not-appear-online/',
+				'type' => 'cta',
+			],
+		];
+
+		$notice             = User::get_user_notices()[ $notice_id ] ?? [];
+		$notice_version     = $notice['meta']['version'] ?? null;
+		$is_version_changed = $this->elementor_version() !== $notice_version;
+
+		if ( $is_version_changed ) {
+			User::set_notice_viewed( $notice_id, false, [ 'version' => $this->elementor_version() ] );
+
+			$excluded_pages = [];
+			$this->print_admin_notice( $options, $excluded_pages );
+		}
+
+		return true;
```

Comment:
Shouldn't it return true only when the notice is displayed? (meaning, inside the condition?) if so, maybe switch the condition to:

`
if ( ! $is_version_changed ) {
	return false;
}
`